woocommerce/woocommerce-android
# Product creation using package photo: Package photo data preview

This PR takes the generated product name, description and scanned keywords from the package photo and transfers the data to the preview screen to complete the AI product creation flow.

[Screen_recording_20231025_001837.webm](https://github.com/woocommerce/woocommerce-android/assets/1522856/7eacdb24-dcac-4680-8d76-993ddec0771d)

**To test:**
1. Go to Products
2. Tap on + FAB
3. Select the AI option
4. Select a package photo
5. Tap on Continue
6. Notice a product preview screen is shown
7. After the product is generated, notice the title and description used is the same as one from the package photo flow

**Note: ** Please merge #10008 first.
